.gitignore

```diff
@@ -203,12 +203,61 @@ Debug
 CMakeScripts
 *.cxx_parameters
 shadow
-
-########
-qucsactivefilter
-*.pro.user
-
-CMakeFiles
-CMakeCache.txt
-*.cmake
-*.depends
+/compile
+/qucs-core/build-aux/ar-lib
+/qucs-core/build-aux/compile
+/qucs-core/build-aux/test-driver
+/qucs-core/build-aux/ylwrap
+/qucs-core/src/components/verilog/constants.vams
+/qucs-core/src/components/verilog/disciplines.vams
+/qucs-core/src/converter/parse_spice.hpp
+/qucs-core/src/converter/parse_vcd.hpp
+/qucs-core/src/parse_citi.hpp
+/qucs-core/src/parse_csv.hpp
+/qucs-core/src/parse_dataset.hpp
+/qucs-core/src/parse_mdl.hpp
+/qucs-core/src/parse_netlist.hpp
+/qucs-core/src/parse_touchstone.hpp
+/qucs-core/src/parse_zvr.hpp
+/qucs/ar-lib
+/qucs/qucs-wideband-matching/CMakeCache.txt
```

Comment:
I am not sure these CMake files should be ignored as such.
If you do out-of-tree build they should not bother you.